lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -651,6 +827,7 @@ def __init__(self, config):
         self.scratch = config['scratch']
         self.callback_url = os.environ['SDK_CALLBACK_URL']
         self.ws_url = config['workspace-url']
+        self.token_user = os.environ['KB_AUTH_TOKEN'].split('client_id=')[1].split('|')[0]
```

Comment:
I think it leaves a potential issue if we change token format in the future. But I not sure a right way to get current user_name.